src/css/abstracts/variables.css

```diff
@@ -12,6 +12,7 @@
 	--beauty-bush: #efb5ba;
```

Comment:
use primary and secondary color class name for a good design system.

Reply:
Thanks Siddant, Changed the variables accordingly will add the primary/secondary colors to the website while working on documentation site